### PR TITLE
Workflow toolbar ability

### PR DESCRIPTION
**Pull request by jianyi-gronk**
image

### CODE REVIEW

File: spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/graph-ui/src/pages/Graph/Design/GraphMap.tsx [hunk 1]
```diff
@@ -110,6 +117,8 @@ const LayoutFlow = memo(() => {
       onNodesDelete={onNodesDeleteHook}
       onEdgesChange={onEdgesChange}
       onContextMenu={onGrapContextMenu}
+      panOnDrag={operationMode === 'hand'}
```
Comment (CoderSerio) [inline spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/graph-ui/src/pages/Graph/Design/GraphMap.tsx:120]: May enum would be a better choice

**Review by CoderSerio — APPROVED** [review]
LGTM

File: spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/graph-ui/src/pages/Graph/Design/GraphMap.tsx (hunk 1 above)
Comment (jianyi-gronk) [reply spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/graph-ui/src/pages/Graph/Design/GraphMap.tsx]: done

**Review by CoderSerio — APPROVED** [review]

**Comment by Helltab** [comment]
LGTM.

**Pull request merged**